horse_racing.rb

```diff
@@ -1,16 +1,21 @@
 # Solution for https://www.codingame.com/ide/puzzle/horse-racing-duals
 class HorseRacing
+  attr_accessor :horses_number
+
+  def initialize
+    @horses_number = gets.to_i
```

Comment:
It's better to use `count` or `size` instead of `number` for variables that express count or size